lib/domain/usecase/room/update_group_chat_interactor.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:dartz/dartz.dart';
+import 'package:fluffychat/app_state/failure.dart';
+import 'package:fluffychat/app_state/success.dart';
+import 'package:fluffychat/domain/app_state/room/update_group_chat_failure.dart';
+import 'package:fluffychat/domain/app_state/room/update_group_chat_loading.dart';
+import 'package:fluffychat/domain/app_state/room/update_group_chat_success.dart';
+import 'package:matrix/matrix.dart';
+
+class UpdateGroupChatInteractor {
+  Stream<Either<Failure, Success>> execute({
+    required Room room,
+    required Client client,
+    Uri? avatarUrl,
```

Comment:
You don't upload avatar inside this interactor?

Reply:
I use it inside chat, same logic update profile.